kanidmd/src/lib/core/https/mod.rs

```diff
@@ -1191,14 +425,17 @@ pub fn create_https_server(
     // tide::log::with_level(tide::log::LevelFilter::Debug);
 
     // Add middleware?
-    tserver.with(tide::log::LogMiddleware::new()).with(
+    tserver.with(tide::log::LogMiddleware::new());
+    tserver.with(
         // We do not force a session ttl, because we validate this elsewhere in usage.
         tide::sessions::SessionMiddleware::new(tide::sessions::MemoryStore::new(), cookie_key)
             .with_cookie_name("kanidm-session")
             .with_same_site_policy(tide::http::cookies::SameSite::Strict),
     );
+    tserver.with(StrictResponseMiddleware::default());
```

Comment:
Why do we use the `Default` constructor? I feel like it would make more sense to do something like `StrictResponseMiddleware {}`, because this shows that it's a ZST and hides less information.

Reply:
because else clippy will whinge if I added a new() -> Self but no default, so I used default instead. I prefered to leave it like this as we may want to add config/customisation later and it's a bit more readable this way.